File: SDKs/googleapis.com/bigtableadmin/v1/typescript/src/sdk/models/shared/split.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// Split
/** 
 * An initial split point for a newly created table.
**/
export class Split extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=key" })
  key?: string;
}

File: SDKs/googleapis.com/bigtableadmin/v1/python/src/sdk/models/shared/split.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class Split:
    r"""Split
    An initial split point for a newly created table.
    """
    
    key: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('key') }})
    
